tests/integration-tests/tests/createami/test_createami/test_build_image/image.config.yaml

```diff
@@ -23,3 +23,6 @@ DeploymentSettings:
         - {{ private_subnet_id }}
         SecurityGroupIds:
         - {{ default_vpc_security_group_id }}
+
+DevSettings:
+    DisableValidateAndTest: False
```

Comment:
Is this necessary?

Reply:
setting this parameter to false make it execute all the Kitchen tests with tag:install_ prefix as part of the Image Builder validation phase

Reply:
can this just be the default value?